plugins/modules/cloud/scaleway/scaleway_database_backup.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/python
+#
+# Scaleway database backups management module
+#
+# Copyright (C) 2020 Guillaume Rodriguez (g.rodriguez@opendecide.com).
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_database_backup
+short_description: Scaleway database backups management module
+author: Guillaume Rodriguez (@guillaume_ro_fr)
+description:
+    - This module manages database backups on Scaleway account U(https://developer.scaleway.com).
+extends_documentation_fragment:
+    - community.general.scaleway
+options:
+  state:
+    description:
+        - Indicate desired state of the database backup.
```

Comment:
It would be good to explain what the different states mean here. To me, it is not clear what all these states mean.